datasets/mnist_usps.py

```diff
@@ -23,32 +27,81 @@ class Dataset(BaseDataset):
     # the cross product for each key in the dictionary.
     # Any parameters 'param' defined here is available as `self.param`.
     parameters = {
-        'n_samples_source, n_samples_target': [(3000, 3000)],
+        #'n_samples_source, n_samples_target': [(3000, 3000)],
+        'n_samples_source, n_samples_target': [(10000, 10000)],
         'source_target': [('MNIST', 'USPS'),
                           ('USPS', 'MNIST')],
         'random_state': [27],
     }
 
-    path_preprocessed_data = "data/digit_preprocessed.pkl"
-    url_preprocessed_data = "https://figshare.com/ndownloader/files/46363525?private_link=f15a4a7c81084815114b"
```

Comment:
Yes, we directly take the raw images instead of preprocessed vectors. We don't need to use feature reduction techniques beforehand since the network will handle that itself.

Reply:
So there is one mnist_usps for shallow methods and one for deep methods?

Reply:
ok I had forgotten, there are mnist_usps and mnist_usps_pca